CHANGELOG.rst

```diff
@@ -1,6 +1,56 @@
 Change log
 ==========
 
+0.2.1 [2024-08-19]
+------------------
+
+Features
+~~~~~~~~
+
+- Added ``bootup_delay`` option to control delay before the agent starts
+  on boot.
+- Restart the agent when ``openwisp-config`` starts or restarts to ensure
+configuration consistency.
+- Added ``htmode`` to wireless interface stats for enhanced monitoring.
+- Added bitrate, quality, and connected AP information for wireless
+  "station" mode.
+- Added support for using the ``cacert`` and ``capath`` configuration
+  options defined in the ``openwisp-config`` configuration file for
+  ``curl`` command.
```

Comment:
I think this is missing:
https://github.com/openwisp/openwrt-openwisp-monitoring/pull/137